# Bioinformatics and Computational Biology

## Alignment

- [bwa](https://github.com/lh3/bwa)
- SRA toolkit
  * [ngs](http://github.com/ncbi/ngs)
  * [ncbi-vdb](http://github.com/ncbi/ncbi-vdb)
  * [sra-tools](http://github.com/ncbi/sra-tools)
- [samtools](https://github.com/samtools/samtools)

## Sequences

- [seqtk](https://github.com/lh3/seqtk)

## Databases

- [KEGG: Kyoto Encyclopedia of Genes and Genomes](http://www.genome.jp/kegg/)
- [dbSNP](http://www.ncbi.nlm.nih.gov/SNP/)
- [SGD](http://www.yeastgenome.org/)
- [PDB](http://www.rcsb.org/pdb/home/home.do)

## File Types

- [Biological data file formats](https://en.wikipedia.org/wiki/Biological_data#Data_File_Formats)
- [FASTA](https://en.wikipedia.org/wiki/FASTA)
- [FASTQ](https://en.wikipedia.org/wiki/FASTQ_format)
- [VCF](https://en.wikipedia.org/wiki/Variant_Call_Format) (variant call format)
- [SAM](https://en.wikipedia.org/wiki/SAM_(file_format)) (sequence alignment map) and BAM (binary alignment map)

## Ontologies

- [EFO](http://www.ebi.ac.uk/efo/)
- [GO](http://geneontology.org/) (Gene Ontology)

## Tutorials
- [Variant calling tutorial](https://wikis.utexas.edu/display/bioiteam/Variant+calling+tutorial)
- [griffithlab/rnaseq_tutorial](https://github.com/griffithlab/rnaseq_tutorial)

## RNASeq

- [Normalization of RNA-seq data using factor analysis of control genes or samples](http://www.nature.com/nbt/journal/v32/n9/full/nbt.2931.html)

## Pipeline Frameworks
- [nextflow](http://www.nextflow.io/)

## Algorithms

- [BWT](https://en.wikipedia.org/wiki/Burrows%E2%80%93Wheeler_transform) (Burrows-Wheeler transform)

## Other Things
- [nucleotides](http://nucleotid.es/)
- [repositive](https://repositive.io/)

## Graph Theory

- [Cluster and Visualize a Subset of Your Neo4j Database Using RNeo4j, igraph, and Alchemy.js.](http://nicolewhite.github.io/2014/07/24/visualize-subset-neo4j-alchemy.html)

## Containers

- [docker-bio-linux8](https://hub.docker.com/r/gawbul/docker-bio-linux8/)
